packages/web-components/fast-foundation/src/tab/stories/tab.register.ts

```diff
@@ -31,6 +31,27 @@ const styles = css`
         color: var(--neutral-foreground-rest);
         fill: currentcolor;
     }
+    slot:not([name]) {
+        display: block;
+        max-width: 100px;
+        overflow: hidden;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+    }
```

Comment:
I know these are just our current testing scenarios, but would prefer we not advocate for a pattern like this. Setting the display value on a slot element which defaults rightly to `display: contents` is quite the antipattern. If we're expecting this to be an intuitive and common pattern, it's not going to be and I think it'll cause confusion.

Reply:
Removed. This was testing and is no longer needed to achieve the same results since the `.content` wrapper is there.